plugins/inventory/terraform.py

```diff
@@ -373,8 +373,10 @@ def main():
     parser.add_argument('--nometa',
                         action='store_true',
                         help='with --list, exclude hostvars')
-    default_root = os.path.abspath(os.path.join(os.path.dirname(__file__),
-                                                '..', '..', ))
+    default_root = os.environ.get('TERRAFORM_STATE_ROOT',
```

Comment:
Also need to make sure this is synchronized to:
https://github.com/CiscoCloud/terraform.py